### PR TITLE
Add Chuwi Hi10 to 60-sensor.hwdb

### PR DESCRIPTION
**Pull request by AlbrechtL**
I own a Chuwi Hi10 tablet PC and I figured out that the accelerometer needs to be rotated.
This PR fixes it.

### CODE REVIEW

File: hwdb/60-sensor.hwdb [hunk 1]
```diff
@@ -160,6 +160,10 @@ sensor:modalias:acpi:BMA250E*:dmi:bvnINSYDECorp.:bvrG1D_S165*:svnilife:pnS165:*
 sensor:modalias:acpi:BOSC0200*:dmi:*:svnHampoo:pnX1D3_C806N:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
 
+# Chuwi Hi10 (CWI1515)
+sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvrP02A_C106.60E:bd03/18/2016:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnHampoo:rnCherryTrailCR:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0
+
```
Comment (keszybz) [inline hwdb/60-sensor.hwdb:166]: This is missing the 9th number.

**Comment by AlbrechtL** [comment]
Oh sorry! It seems that I missed the number while using the touch pad for copying :-(

Done in https://github.com/systemd/systemd/pull/13351/commits/02fcc68ffba8450a3754d4805181191d0e82d7aa

File: hwdb/60-sensor.hwdb [hunk 2]
```diff
@@ -160,6 +160,10 @@ sensor:modalias:acpi:BMA250E*:dmi:bvnINSYDECorp.:bvrG1D_S165*:svnilife:pnS165:*
 sensor:modalias:acpi:BOSC0200*:dmi:*:svnHampoo:pnX1D3_C806N:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
 
+# Chuwi Hi10 (CWI1515)
+sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvrP02A_C106.60E:bd03/18/2016:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnHampoo:rnCherryTrailCR:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:
```
Comment (keszybz) [inline hwdb/60-sensor.hwdb]: I'm pretty sure this line needs to be trimmed. It should not include the bd at least.
Maybe 
`sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvrP02A_C106.60E:*:svnDefaultstring:pnDefaultstring:*`.

Comment (AlbrechtL) [reply hwdb/60-sensor.hwdb]: Thanks for pointing this out. I will test it tomorrow evening.

Comment (AlbrechtL) [reply hwdb/60-sensor.hwdb]: The trimmed identification line is working.

Added in https://github.com/systemd/systemd/pull/13351/commits/ac820765855a4f93730ce7bf4a6cfb0db60be4d3.

**Pull request merged**